boards/xtensa/m5stack_core2/m5stack_core2.dts

```diff
@@ -154,7 +159,7 @@
 		};
 	};
 
-	ft5336@38 {
+	ft5336_touch: ft5336@38 {
```

Comment:
Could you drop the `kscan-input` down below in the process? That was a temporary thing and can go now that we are pointing boards to the native driver.

Reply:
I was expecting `kscan-input` is still needed for non-lvgl applications. Can you please explain in more detail what you mean with temporary thing? Is there an alternative for non-lvgl applications?

Reply:
The kscan-input in the context of lvgl was used for compatibility. All touch drivers previously used the keyscan API. With the advent of the input subsystem touch drivers were converted to it. As a compatibility layer the kscan-input was added. Eventually we want to move away from using kscan nodes for lvgl pointer devices, since there is now the `lvgl_pointer_input` node.

Reply:
Right, if the application still needs it then it can go in an overlay in the application itself.